src/praisonai-agents/praisonaiagents/agent/tool_execution.py

```
@@ -1839,6 +1839,35 @@ def _execute_tool_with_circuit_breaker(self, function_name, arguments):
             raise last_exception
         return {"error": "Maximum retry attempts exceeded"}
 
+    @staticmethod
+    def _remove_circuit_breaker(breaker_name):
+        """Remove a circuit breaker entry from the process-global registry."""
+        try:
+            from ..tools.circuit_breaker import _get_global_registry
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)